packages/docs/docs/100-getting-started.md

```diff
@@ -27,35 +166,166 @@ npx @synthql/cli generate \
     --out=./src/generated
 ```
 
-This will generate a file at `src/generated/synthql/db.ts`.
+This will generate a types file at `src/generated/db.ts`, a schema definitions file at `src/generated/schema.ts` and an index file that exports both at `src/generated/index.ts`.
 
 ## Write your first query
 
 ```ts
-import { from } from 'src/db';
+// src/queries.ts
+import { Query } from '@synthql/queries';
+import { DB, from } from 'src/generated';
 
-const findUserByIds = (ids: string[]) => {
+const findUserByIds = (ids: string[]): Query<DB> => {
```

Comment:
If you add Query<DB> you break the type safety, becuase the QueryResult expects a query with literals

```suggestion
const findUserByIds = (ids: string[]) => {
```